cgk_calcium_tools/filehandler.py

```diff
@@ -289,33 +290,50 @@ def de_interleave(self, overwrite: bool = False) -> None:
             if len(focus) > 1:  # has multiplane
                 existing_files = []
                 for sp_file in planes_fs:
+                    json_file = os.path.splitext(sp_file)[0] + ".json"
                     if os.path.exists(sp_file):
                         if overwrite:
                             os.remove(sp_file)
+                            if os.path.exists(json_file):
+                                os.remove(json_file)
                         else:
-                            existing_files.append(sp_file)
+                            if same_json_or_remove(
+                                parameters={
+                                    "main_file": main_file,
+                                    "planes_fs": planes_fs,
+                                    "focus": focus,
```

Comment:
following the other functions, it's better to try to use same keys as function arguments. In this case: input_movie_files, output_movie_files, in_efocus_values